pkg/lib/journal.js

```diff
@@ -211,7 +211,7 @@ journal.printable = function printable(value) {
     else if (typeof (value) == "string")
         return value;
     else if (value.length !== undefined)
-        return cockpit.format(_("[$0 bytes of binary data]"), value.length);
+        return new TextDecoder().decode(new Uint8Array(value));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18603-20230414-084535-cb6b8350-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)